ods_ci/tests/Resources/Page/ODH/Prometheus/Prometheus.robot

```diff
@@ -121,6 +121,9 @@ Alert Should Be Firing    # robocop: disable:too-many-calls-in-keyword
                     END
                 END
             END
+            IF    ${alert_found}
+                Exit For Loop
+            END
```

Comment:
why not keep checking for other alerts?

Reply:
This keyword `Alert Should Be Firing`  is meant to find if a particular alert is firing in an alert_group.  This change is to address a bug when, after having found the  particular alert, in the next loop iteration there is an alert group with the same name and the value of ${alert_found} is set back to False.